e2e_tests/visual_regression_test.py

```diff
@@ -1,25 +1,29 @@
-import re
-
-from playwright.sync_api import expect
+import optuna
 from playwright.sync_api import Page
+import pytest
 
 
-def test_homepage_has_Playwright_in_title_and_get_started_link_linking_to_the_intro_page(
+@pytest.mark.parametrize("study_id", range(10))
+def test_study_list(
+    study_id: int,
     page: Page,
+    dummy_storage: optuna.storages.InMemoryStorage,
+    port: int,
+    server: None,
```

Comment:
Does it sound reasonable to return the base url like "http://localhost:{port}/dashboard/", then we can remove a `port` argument.

Reply:
Fixed.